### ISSUE TITLE
log path flag not available

### ISSUE THREAD

**Issue by BaptisteGerondeau** [Linaro/mr-provisioner-role#3]
https://github.com/Linaro/mr-provisioner-role/blob/529d4780dd57758ef4a3ce39edf0f685934f89b9/templates/mr-provisioner-tftp.service.j2#L8

On a fresh installation I had problems with this peculiar flag, it seems -log-path is not recognized and the service did not launch.
Would it be an external library that needs to be installed ?

**Comment by niedbalski** [comment]
Nope, the -log-path flag wasn't merged upstream, instead we are relying on the stderr/stdout from systemd. So I think this flag should be removed from the template altogether.
